### PR TITLE
feat: Select multiple controls using rectangle area

### PR DESCRIPTION
**Pull request by rzaaeeff**

### CODE REVIEW

**Comment by rzaaeeff** [comment]
@adilek please, check this out.

**Comment by rzaaeeff** [comment]
[Select several components using rectangle area.](https://trello.com/c/LswWE9Fj/1-select-several-components-using-rectangle-area)

**Comment by rzaaeeff** [comment]
[Select several components using rectangle area.](https://trello.com/c/LswWE9Fj/1-select-several-components-using-rectangle-area)

File: app/app/Board.js [hunk 1]
```diff
@@ -254,6 +275,15 @@ export class Board {
      */
     onMouseUp(event) {
```
Comment (adilek) [inline app/app/Board.js]: In my opinion you should select/unselect during mouse move not mouseUp.
Just a UX example select several files in Windows explorer. The files are being selected/unselectd on mousemove event. What do you think?

**Review by adilek — CHANGES_REQUESTED** [review]

**Comment by rzaaeeff** [comment]
@adilek I think it would be cool, too.

I've tried it, it slows drawing process (frame rate drops) when there are many controls in board.

**Comment by adilek** [comment]
@rzaaeeff, 
Thanks for clarification. I think lets make with mouse move even there are framedrops or other performance issues. Lets think about it later. First need to focus on UX then performance in my opinion.

**Review by adilek — APPROVED** [review]

**Comment by rzaaeeff** [comment]
@adilek thank you.

**Pull request merged**